src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -262,17 +262,14 @@ def test_graph_group_scenario(self):
         account_info = self.cmd('account show').get_output_in_json()
         if account_info['user']['type'] == 'servicePrincipal':
             return  # this test delete users which are beyond a SP's capacity, so quit...
-        upn = account_info['user']['name']
-        domain = upn.split('@', 1)[1]
         self.kwargs = {
             'user1': 'deleteme1',
             'user2': 'deleteme2',
-            'domain': domain,
+            'domain': 'azuresdkteam.onmicrosoft.com',
```

Comment:
Let's not hard code it. But create a new user in the `azuresdkteam.onmicrosoft.com` tenant with **global admin** permission to run this test again.